private/bufpkg/bufimage/bufimagemodify/bufimagemodify.go

```diff
@@ -169,34 +197,34 @@ func GoPackageImportPathForFile(imageFile bufimage.ImageFile, importPathPrefix s
 //  * If the resulting abbreviation is 1 character, add "XX".
 //  * If the resulting abbreviation is "GPB", change it to "GPX".
 //    "GPB" is reserved by Google for the Protocol Buffers implementation.
-func ObjcClassPrefix(sweeper Sweeper) Modifier {
-	return objcClassPrefix(sweeper)
+func ObjcClassPrefix(sweeper Sweeper, overrides map[string]string) Modifier {
```

Comment:
Something feels wrong about passing these as required parameters, I can't pin it though

Reply:
Yea this one's tough. The alternatives are:
1. Add a `ModifierOption` that is shared amongst all `Modifier` constructors. But as soon as one option should belong to a specific `Modifier` and not others, it breaks down.
2. Add a new `<ModifierName>Option` for each `Modifier`. But this is really verbose.

Given that the `nil` value for `overrides` is actually safe in all of these cases, it feels like the least bad option, especially because it keeps the boilerplate to a minimum. There might be other viable alternatives I'm not seeing though.